content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+is_beta: true
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] provides infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. Once you are enrolled in the private beta, the [Container Images][2] page and banner with additional capabilities are viewable. 
+
+{{< img src="security/infrastructure_vulnerability/banner.png" alt="Banner Page" width="100%">}}
+
+## Review the health of your container images
+
+Infrastructure Vulnerability Management offers vulnerabilities in the Container Images on your hosts.
+View your container images on the [Container Images][2] page. Sort images by the number of active containers (column name `CONTAINERS`), observe the number of vulnerabilities that exist in the container image (column name `VULNERABILITIES`).
+View additional details about the vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.
```

Comment:
View additional details about any vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.